### DISCUSSION TITLE
Putting tic80 program on web

### DISCUSSION THREAD

**Discussion by fumiinaki** [nesbox/TIC-80#1449] [Q&A]
I created index.html, tic80.wasm, cart.ti and tic80.js by export html command.
How can I see the program running.
I tried to run index.html on the local drive but it only shows CLICK and nothing happens even you click it.
I want to put it on my we page created by wordpress.
Please help me

**Comment by nesbox** [comment]
It doesn't work locally, you have to upload it to the webserver
